sycl/include/sycl/detail/pi.h

```diff
@@ -331,7 +332,8 @@ typedef enum {
   PI_QUEUE_INFO_DEVICE_DEFAULT = 0x1095,
   PI_QUEUE_INFO_PROPERTIES = 0x1093,
   PI_QUEUE_INFO_REFERENCE_COUNT = 0x1092,
-  PI_QUEUE_INFO_SIZE = 0x1094
+  PI_QUEUE_INFO_SIZE = 0x1094,
+  PI_QUEUE_INFO_STATUS = 0x1096
```

Comment:
Maybe it would be better if we used ONEAPI is name and more different number? Something like:
`PI_EXT_ONEAPI_QUEUE_INFO_STATUS` and the number `0x2096`.

Reply:
Good call, renamed and changed the number.